security/rbac/scripts/enable-rbac-connect.sh

```diff
@@ -93,57 +93,62 @@ confluent iam rolebinding list --principal User:$USER_ADMIN_CONNECT --kafka-clus
-confluent iam rolebinding create --principal User:$USER_CLIENT_A --role ResourceOwner --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+echo -e "\n# Grant principal User:$USER_CONNECTOR_SUBMITTER the ResourceOwner role to Connector:$CONNECTOR_NAME"
+echo "confluent iam rolebinding create --principal User:$USER_CONNECTOR_SUBMITTER --role ResourceOwner --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID"
+confluent iam rolebinding create --principal User:$USER_CONNECTOR_SUBMITTER --role ResourceOwner --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
 
-echo -e "\n# List the role bindings for the principal User:$USER_CLIENT_A to the Connect cluster"
-echo "confluent iam rolebinding list --principal User:$USER_CLIENT_A --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID"
-confluent iam rolebinding list --principal User:$USER_CLIENT_A --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+echo -e "\n# List the role bindings for the principal User:$USER_CONNECTOR_SUBMITTER to the Connect cluster"
+echo "confluent iam rolebinding list --principal User:$USER_CONNECTOR_SUBMITTER --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID"
+confluent iam rolebinding list --principal User:$USER_CONNECTOR_SUBMITTER --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
 
-echo -e "\n# Grant principal User:$USER_CLIENT_A the ResourceOwner role to Topic:$TOPIC2_AVRO"
-echo "confluent iam rolebinding create --principal User:$USER_CLIENT_A --role ResourceOwner --resource Topic:$TOPIC2_AVRO --kafka-cluster-id $KAFKA_CLUSTER_ID"
-confluent iam rolebinding create --principal User:$USER_CLIENT_A --role ResourceOwner --resource Topic:$TOPIC2_AVRO --kafka-cluster-id $KAFKA_CLUSTER_ID
+echo -e "\n# Grant principal User:$USER_CONNECTOR_TASK the ResourceOwner role to Topic:$TOPIC2_AVRO"
+echo "confluent iam rolebinding create --principal User:$USER_CONNECTOR_TASK --role ResourceOwner --resource Topic:$TOPIC2_AVRO --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:$USER_CONNECTOR_TASK --role ResourceOwner --resource Topic:$TOPIC2_AVRO --kafka-cluster-id $KAFKA_CLUSTER_ID
 
-echo -e "\n# List the role bindings for the principal User:$USER_CLIENT_A to the Kafka cluster"
-echo "confluent iam rolebinding list --principal User:$USER_CLIENT_A --kafka-cluster-id $KAFKA_CLUSTER_ID"
-confluent iam rolebinding list --principal User:$USER_CLIENT_A --kafka-cluster-id $KAFKA_CLUSTER_ID
+echo -e "#\n Grant principal User:$USER_CONNECTOR_TASK the DeveloperRead role to Group:console-consumer- prefix"
```

Comment:
Might be good to note here that this isn't required to run connect with RBAC and is just included here so that the console consumer can be run later on with this principal.

Reply:
@C0urante great catch, thanks for pointing this out.